front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/V2PipelineTemplateController.java

```diff
@@ -186,22 +186,23 @@ List<String> getDependentConfigs(String templateId) {
     List<String> dependentConfigIds = new ArrayList<>();
 
     String prefixedId = SPINNAKER_PREFIX + templateId;
+
     pipelineDAO.all().stream()
         .filter(pipeline -> pipeline.getType() != null && pipeline.getType().equals(TYPE_TEMPLATED))
         .forEach(
             templatedPipeline -> {
               String source;
               try {
-                TemplateConfiguration config =
-                    objectMapper.convertValue(
-                        templatedPipeline.getConfig(), TemplateConfiguration.class);
 
-                source = config.getPipeline().getTemplate().getSource();
+                V2TemplateConfiguration config =
+                    objectMapper.convertValue(templatedPipeline, V2TemplateConfiguration.class);
+                source = config.getTemplate().getReference();
               } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
We don't typically print to standard out in shipping code.  If there's something to log here, please do.

Reply:
This is removed.